README.md

```diff
@@ -166,6 +166,91 @@ optional `--wait --timeout 15m` parameters in a deployment pipeline to see if th
 To deploy Magento to different environments (develop, staging, production) it is recommended to create a `values_*.yaml`
 for each environment and tune the resource limits and configuration values of the services.
 
+## Docker Desktop Example
+The file `values_docker.yaml` is a shorter version of `values.yaml` which contains all the changes necessary 
```

Comment:
@phoenix-henri `values_docker.yaml` inherits all valeus from `values.yaml`. Therefore it only changes values necessary to run the Helm chart in the Docker Desktop K8S environment.